docs/reference/hooks.rst

```diff
@@ -477,6 +477,15 @@ Hooks for customising the way users are directed through the process of creating
     def do_after_page_create(request, page):
         return HttpResponse("Congrats on making content!", content_type="text/plain")
 
+  If you set attributes on a ``Page`` object, you should also call ``save_revision()``, since the edit and index view pick up their data from the revisions table rather than the actual saved page record.
+  
+  .. code-block:: python
+  
+      @hooks.register('after_create_page')
+      def set_attribute_after_page_create(request, page)
+         page.title = 'Persistent Title'
+         page.save_revision()
+         page.save()
```

Comment:
No need to call `.save()` here since the new title will already be saved. If you meant to publish the changes, you should call `.publish()` on the saved revision: `page.save_revision().publish()`

Reply:
Ah, I guess going for the `title` attribute as an example was the wrong choice. I actually tested this with an attribute set on my sub-classed model `MyPage` (i.e. inheriting from `Page`). There, my attribute does not get set if I don't call `.save()` on top of `.save_revision()`.